spec/rubocop/cop/rails/squished_heredocs_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Rails::SquishedHeredocs, :config do
+  subject(:cop) { described_class.new(config) }
+
+  it 'registers an offense' do
+    expect_offense(<<~RUBY)
+      <<~SQL
+      ^^^^^^ Use `<<~SQL.squish` instead of `<<~SQL`.
+        SELECT * FROM table;
+      SQL
```

Comment:
I'm not sure there should be an offense here: it's a single line squiggly heredoc, `squish` will have no effect.

I'd even say that ideally, this cop would remove `squish` from single line heredocs, converting them to be squiggly if necessary.

Reply:
Done